packages/contracts-bedrock/contracts/universal/StandardBridge.sol

```diff
@@ -372,12 +503,165 @@ abstract contract StandardBridge {
+
+        MESSENGER.sendMessage{ value: _amount }(
+            BridgeConstants.BIT_TX,
+            _amount,
+            address(OTHER_BRIDGE),
+            abi.encodeWithSelector(
+                this.finalizeBridgeBITWithdraw.selector,
+                _remoteToken,
+                _localToken,
+                _from,
+                _to,
+                _amount,
+                _extraData
+            ),
+            _minGasLimit
+        );
+
+
+    }
+
+    /**
+ * @notice Sends ERC20 tokens to a receiver's address on the other chain.
+     *
+     * @param _localToken  Address of the ERC20 on this chain.
+     * @param _to          Address of the receiver.
+     * @param _amount      Amount of local tokens to deposit.
+     * @param _minGasLimit Minimum amount of gas that the bridge can be relayed with.
+     * @param _extraData   Extra data to be sent with the transaction. Note that the recipient will
+     *                     not be triggered with this data, but it will be emitted and can be used
+     *                     to identify the transaction.
+     */
+    function _initiateBridgeBITDeposit(
+        address _localToken,
+        address _from,
+        address _to,
+        uint256 _amount,
+        uint32 _minGasLimit,
+        bytes memory _extraData
+    ) internal {
+
+        IERC20(_localToken).safeTransferFrom(_from, address(this), _amount);
+        deposits[_localToken][address(0) ] = deposits[_localToken][address(0)] + _amount;
+
+        // Emit the correct events. By default this will be ERC20BridgeInitiated, but child
+        // contracts may override this function in order to emit legacy events as well.
+        _emitBITBridgeInitiated(_localToken, address(0) , _from, _to, _amount, _extraData);
+
+        MESSENGER.sendMessage(
+            BridgeConstants.BIT_TX,
+            _amount,
+            address(OTHER_BRIDGE),
+            abi.encodeWithSelector(
+                this.finalizeBridgeBITDeposit.selector,
+                _from,
+                _to,
+                _amount,
+                _extraData
+            ),
+            _minGasLimit
+        );
+    }
+
+    /**
+ * @notice Sends ETH tokens to a receiver's address on the other chain.
+     *
+     * @param _localToken  Address of the ETH on this chain.
+     * @param _remoteToken Address of the ETH on other chain.
+     * @param _from          Address of the sender.
+     * @param _to          Address of the receiver.
+     * @param _amount      Amount of local tokens to deposit.
+     * @param _minGasLimit Minimum amount of gas that the bridge can be relayed with.
+     * @param _extraData   Extra data to be sent with the transaction. Note that the recipient will
+     *                     not be triggered with this data, but it will be emitted and can be used
+     *                     to identify the transaction.
+     */
+    function _initiateBridgeETHWithdrawal(
+        address _localToken,
+        address _remoteToken,
```

Comment:
`_remoteToken` must be zero address on L1. Can we just leave out this parameter?